contrib/swtpm/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "swtpm"
+pkgver = "0.8.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--with-gnutls"]
+configure_env = {"NOCONFIGURE": "1"}
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "bash",
+    "expect",
+    "gawk",
+    "gmake",
+    "gnutls-progs",
+    "libtool",
+    "pkgconf",
+    "socat",
+]
+makedepends = [
+    "gnutls-devel",
+    "json-glib-devel",
+    "libseccomp-devel",
+    "libtasn1-devel",
+    "libtpms-devel",
+    "linux-headers",
+    "openssl-devel",
+]
+checkdepends = [
+    "iproute2",
+    "trousers",
+]
+depends = ["gnutls-progs"]
+pkgdesc = "TPM emulator with socket, character device, and Linux CUSE interface"
+maintainer = "cesorious <cesorious@gmail.com>"
+license = "BSD-3-Clause"
+url = "https://github.com/stefanberger/swtpm"
+source = (
+    f"https://github.com/stefanberger/swtpm/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "7bba52aa41090f75087034fac5fe8daed10c3e7e7234df7c9558849318927f41"
+
+
+def post_install(self):
+    self.install_license("LICENSE")
+
+
+@subpackage("libswtpm")
+def _libs(self):
+    return ["usr/lib/swtpm/*.so.*"]
```

Comment:
```suggestion
    return self.default_libs()
```

Reply:
this doesn't seem to work because the libs are in a subdirectory, at least it fails with
`=> libswtpm-0.8.1-r0: ERROR: take(): path 'usr/lib/lib*.so.[0-9]*' did not match anything`

Reply:
oh right i missed the subdir